packages/annotation-toolkit/src/helpers.js

```diff
@@ -2,6 +2,13 @@ export const frameToMs = (frame, fps) => (frame * 1000) / fps;
 
 export const msToFrame = (ms, fps) => Math.round((ms * fps) / 1000);
 
-export const requestQueue = (layer) => {
-  return ["layers", layer, "data", "requests"];
+export const requestsPathFor = (layerId) => {
+  return ["layers", layerId, "requests"];
 };
+
+export const dataPathBuilderFor = (layerId) => (...args) => [
```

Comment:
Worth adding a comment explaining this const?

Reply:
added some comments!